examples/research_projects/diffusion_dpo/train_diffusion_dpo_sdxl.py

```diff
@@ -423,6 +423,11 @@ def parse_args(input_args=None):
     parser.add_argument(
         "--enable_xformers_memory_efficient_attention", action="store_true", help="Whether or not to use xformers."
     )
+    parser.add_argument(
+        "--is_turbo",
+        action="store_true",
+        help=("Use if tuning SDXL Turbo instead of SDXL"),
```

Comment:
nit: if we remove the help statement out of the brackets and remove the comma it probably will go to line above.